chore(renew certif): close issue 2950 (#177)
jenkins-infra/status: content/issues/2022-05-31-update-centernot-updated.md

---
title: Update Center not updated with new released plugins
date: 2022-05-31T12:00:00-00:00
resolved: true

> resolvedWhen: 2022-06-01T08:10:00-00:00

Reviewer: ```suggestion
resolvedWhen: 2022-06-01T07:34:00-00:00
```

# Possible severity levels: down, disrupted, notice
severity: disrupted
affected:
  - updates.jenkins.io
section: issue
---

[Close Issue]

> the new certificate had been issued and store within the trusted controller. Both the crawler an the update-center job run again smoothly.

Reviewer 2: ```suggestion
the new certificate had been issued and stored within the trusted controller. Both the crawler and the update-center jobs run smoothly again.
```

> the new certificate had been issued and stored within the trusted controller. Both the crawler and the update-center jobs run smoothly again.

Reviewer: ```suggestion
the new certificate had been issued and stored within the trusted controller. Both the crawler and the update-center jobs run smoothly again since 07:34am UTC.

The plugin "gap" was filled at 09:47am UTC as per <https://github.com/jenkins-infra/helpdesk/issues/2950#issuecomment-1143374848>.

```
